templates/emails/enroll_email_allowedsubject.txt

```diff
@@ -0,0 +1,5 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+${_("""Invitation to register for Enthought's "{course_name}" """.format(
+    course_name=course.display_name_with_default
+))}
```

Comment:
Shouldn't this file be `enroll_email_allowed_subject.txt` ? The last underscore missing before `subject.txt` seems to be a typo?